src/Keboola/DbExtractor/TableResultFormat/Table.php

```diff
@@ -64,7 +64,14 @@ public function setRowCount(?int $rowCount): self
 
     public function addColumn(TableColumn $column): self
     {
-        $this->columns[] = $column;
+        $columnKey = $column->getOrdinalPosition() ?: $column->getSanitizedName();
+        if (isset($this->columns[$columnKey])) {
+            $oldColumn = $this->columns[$columnKey];
+            if ($oldColumn->isPrimaryKey()) {
+                $column->setPrimaryKey(true);
+            }
```

Comment:
I believe this will be the case for any index
Please check out how it was working in the previous tag of mssql extractor.  https://github.com/keboola/db-extractor-mssql/blob/5.1.2/src/Keboola/DbExtractor/Extractor/MetadataProvider.php#L242
If the column already existed in the array then it skipped the main entries, but added other info provided by the index object (FK info for example)

Reply:
i added other info for merge tho same columns